eclair-core/src/main/scala/fr/acinq/eclair/crypto/Sphinx.scala

```diff
@@ -410,6 +411,19 @@ object Sphinx extends Logging {
       BlindedRoute(publicKeys.head, blindingKeys.head, blindedHops)
     }
 
+    /**
+     * Creates a direct blinded route with a single node
+     *
+     * @param sessionKey this node's session key.
+     * @param nodeId     public key of the target node, which is also the introduction point.
+     * @param selfId     payloads that should be encrypted for each node on the route.
+     * @return a blinded route.
+     */
+    def createDirect(sessionKey: PrivateKey, nodeId: PublicKey, selfId: ByteVector): BlindedRoute = {
```

Comment:
I don't think this function belongs here, it's more an application-level utility. Maybe it makes more sense in `Bolt12Invoice.scala` (or maybe it's not even need yet, it will be added once we add code to find suitable blinded paths)?